src/locales/scripts/json-to-pot.js

```diff
@@ -2,8 +2,21 @@ const json = require('../en.json')
 const fs = require('fs')
 const { createPotFile } = require('./json-pot-helpers')
 
+const matchPotCreationDate = /^"POT-Creation-Date: .*\\n"$/gm
+
 try {
   const fileName = process.cwd() + '/src/locales/po-files/openverse.pot'
+  const existingPotFile = String(fs.readFileSync(fileName)).replace(
+    matchPotCreationDate,
+    ''
+  )
+  const potFile = createPotFile(json)
+
+  if (existingPotFile === potFile.replace(matchPotCreationDate, '')) {
+    console.log('No change detected in pot file, skipping write')
+    return process.exit(0)
+  }
```

Comment:
very clever fix for easy diffing of the POT files.

Reply:
Wouldn't we need the replace on left side as well?

Reply:
It's on line 9.

Reply:
The date is stripped from the existing POT file in the preceding lines 9-12

Reply:
Ah cool, thanks. Don't know how I missed that.